include/neml2/models/solid_mechanics/PerfectlyPlasticYieldFunction.h

```diff
@@ -1,40 +0,0 @@
-// Copyright 2023, UChicago Argonne, LLC
```

Comment:
Same thing.  I liked having the nicely named objects around.

Reply:
They are back :)